debian/endless-ca-cert.postinst

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+set -e
+
+CONF_FILE=/etc/ca-certificates.conf
+CONF_FILE_NEW="$CONF_FILE".dpkg-new
+CONF_FILE_OLD="$CONF_FILE".dpkg-old
+CA_CONF=endless/endless-ca.crt
+
+if [ "$1" = configure ]; then
```

Comment:
Maybe put configure in quotes - I know it's not necessary but it looks like a varname without them

Reply:
OK